src/client/java/teammates/client/scripts/ListActiveInstructors.java

```diff
@@ -0,0 +1,52 @@
+package teammates.client.scripts;
+
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.ZoneOffset;
+import java.util.HashSet;
+import java.util.Set;
+
+import teammates.client.remoteapi.RemoteApiClient;
+import teammates.storage.entity.FeedbackSession;
+
+/**
+ * Script to generate emails of active instructors within a period.
+ */
+public class ListActiveInstructors extends RemoteApiClient {
+    @Override
+    protected void doOperation() {
+        //2010/01/01
+        int startDate = 1;
+        int startMonth = 1;
+        int startYear = 2010;
+        //2021/02/01
+        int endDate = 1;
+        int endMonth = 3;
+        int endYear = 2021;
+
+        LocalDate startPoint = LocalDate.of(startYear, startMonth, startDate);
+        LocalDate endPoint = LocalDate.of(endYear, endMonth, endDate);
+        long startTimeInMilli = startPoint.atStartOfDay().toInstant(ZoneOffset.UTC).toEpochMilli();
+        long endTimeInMilli = endPoint.atStartOfDay().toInstant(ZoneOffset.UTC).toEpochMilli();
+        Instant startTime = Instant.ofEpochMilli(startTimeInMilli);
+        Instant endTime = Instant.ofEpochMilli(endTimeInMilli);
+
+        Set<String> activeInstructorEmails = new HashSet<>();
+
+        ofy().load().type(FeedbackSession.class)
```

Comment:
almost there, you have to use a field given in the class as a filter (and the field has to be indexed)

```java
ofy().load().type(FeedbackSession.class)
        .filter("startTime >=", startTime)
        .filter("startTime <=", endTime).project().forEach(feedbackSession -> {
            activeInstructorEmails.add(feedbackSession.getCreatorEmail());
        })
```

Let's use `startTime`